src/components/sortingFunctions.js

```diff
@@ -0,0 +1,40 @@
+export const getShoppingItemBackgroundStyles = (
```

Comment:
Really like/appreciate that these functions/logic are separated out from the main components. Super helpful for readability and getting oriented in the code itself.

Great work!